### PR TITLE
bump: Gradle to the latest

### PR DESCRIPTION
**Pull request by hsinn0**
[#186417987]

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -202,11 +202,11 @@ fi
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
 DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
 
-# Collect all arguments for the java command;
-#   * $DEFAULT_JVM_OPTS, $JAVA_OPTS, and $GRADLE_OPTS can contain fragments of
-#     shell script including quotes and variable substitutions, so put them in
-#     double quotes to make sure that they get re-expanded; and
-#   * put everything else in single quotes, so that it's not re-expanded.
+# Collect all arguments for the java command:
+#   * DEFAULT_JVM_OPTS, JAVA_OPTS, JAVA_OPTS, and optsEnvironmentVar are not allowed to contain shell fragments,
```
Comment (Tallicia) [inline gradlew:206]: JAVA_OPTS is duplicated here.

Comment (hsinn0) [reply gradlew:206]: The files have been auto-generated by gradle and not meant for manual editing. So I would ignore that.

File: gradlew [hunk 2]
```diff
@@ -145,15 +145,15 @@ if ! "$cygwin" && ! "$darwin" && ! "$nonstop" ; then
...
+        # shellcheck disable=SC2039,SC3045
         MAX_FD=$( ulimit -H -n ) ||
             warn "Could not query maximum file descriptor limit"
     esac
     case $MAX_FD in  #(
       '' | soft) :;; #(
       *)
         # In POSIX sh, ulimit -n is undefined. That's why the result is checked to see if it worked.
-        # shellcheck disable=SC3045
+        # shellcheck disable=SC2039,SC3045
```
Comment (Tallicia) [inline gradlew:156]: What is this SC2039?

**Review by Tallicia — APPROVED** [review]
LGTM

File: gradlew (hunk 2 above)
Comment (hsinn0) [reply gradlew:156]: Don't know.

**Pull request merged**